src/index.html

```diff
@@ -5,9 +5,291 @@
+          <div class="person">
+            <img
+              src="images/people/1.jpg"
+              alt="Tim Cook"
+              class="person__photo"
+            >
+            <h3 class="person__name">Tim Cook</h3>
+            <p class="person__position">CEO</p>
+            <p class="person__text">
+              Nice work, Certificates National Association of Realtors (USA).
+            </p>
+          </div>
+
+          <div class="person">
+            <img
+              src="images/people/2.jpg"
+              alt="Katherine Adams"
+              class="person__photo"
+            >
+            <h3 class="person__name">Katherine Adams</h3>
+            <p class="person__position">Senior Vice President</p>
+            <p class="person__text">
+              My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.
+            </p>
+          </div>
+
+          <div class="person">
+            <img
+              src="images/people/3.jpg"
+              alt="Jonathan Ive"
+              class="person__photo"
+            >
+            <h3 class="person__name">Jonathan Ive</h3>
+            <p class="person__position">Chief Design Officer</p>
+            <p class="person__text">
+              When my lovely valley of steam rises and half-day sun is above an impermeable.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section">
+        <h2 class="heading section__heading">Contact Us<span class="heading__crown"></span></h2>
+        <div class="contact">
+
+          <div class="contact__text">
+            <p>
+              <a href="mailto:hello@miami.com" class="link">Hello@miami.com</a><br>
+              <a href="tel:+1234555-55-55" class="link">+1 234 555-55-55</a>
+            </p>
+            <p>
+              <a href="https://goo.gl/maps/s3srqcmmUMtyKy3K9" class="link">400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401</a>
+            </p>
+          </div>
+
+          <form action="" class="form contact__form">
+            <label for="contact_name" class="form__item">
+              <input
+                type="text"
+                name="name"
+                id="contact_name"
+                placeholder="Name"
+                class="input form__input"
+              >
+            </label>
+            <label for="contact_email" class="form__item">
+              <input
+                type="text"
+                name="email"
+                id="contact_email"
+                placeholder="Email"
+                class="input form__input"
+              >
+            </label>
+            <label for="message">
+              <textarea
+                name="message"
+                id="message"
+                placeholder="Message"
+                class="input form_input input__textarea"
```

Comment:
```suggestion
                class="input form_input input__textarea"
```
тут не понятно что есть модификатор, что эелемент